### DISCUSSION TITLE
Known working configuration for Perlmutter? Mpich

### DISCUSSION THREAD

**Discussion by etpalmer63** [nwchemgit/nwchem#774] [Q&A] [answered]
Hi, 

I was wondering if there is a known working configuration for installing and running on Perlmutter at NERSC? Is anyone currently doing this? Is there a recommended version to run? Also, does NWChem run with MPICH? 

Thanks!

Erik

**Comment by dmejiar** [comment, marked as answer]
Hi @etpalmer63 , if you do not have a locally modified version, then you could use shifter images and avoid the compilation step altogether. Please look at [NWChem](https://docs.nersc.gov/applications/nwchem/).
You will find an example SLURM script for PERLMUTTER for version 7.0.2

And yes, NWChem can run with MPICH

**Comment by jeffhammond** [comment]
It works fine.  You can use Intel, GCC, and NVHPC compilers at least.  I do not know if Cray Fortran compiles NWChem correctly or not.

The only interesting choice is `ARMCI_NETWORK`.  You should probably use `MPI-PR` although I regularly use `ARMCI` (i.e. ARMCI-MPI).  Other ARMCI options are unlikely to work better than these.

You should always use Cray MPI, which is based on MPICH.

**Comment by etpalmer63** [comment]
Thanks for the fast responses! I was able to confirm that the script with Shifter for Perlmutter GPU was working. 

Thanks again for everyone's help!
